app/helpers/Evaluation/EvaluationResults/TaskResults/TaskResult.php

```diff
@@ -49,9 +51,12 @@ public function __construct(array $data) {
     $this->status = $data[self::STATUS_KEY];
 
     if (isset($data[self::OUTPUT_KEY])) {
-      $this->output = $data[self::OUTPUT_KEY];
-    } else {
-      $this->output = "";
+      if (isset($data[self::OUTPUT_KEY][self::OUTPUT_STDOUT_KEY])) {
+        $this->output .= $data[self::OUTPUT_KEY][self::OUTPUT_STDOUT_KEY];
+      }
+      if (isset($data[self::OUTPUT_KEY][self::OUTPUT_STDERR_KEY])) {
+        $this->output .= $data[self::OUTPUT_KEY][self::OUTPUT_STDERR_KEY];
+      }
```

Comment:
We should probably make sure that there is at least a newline between stdout and stderr.

Reply:
![Come on](https://media.giphy.com/media/MwOuiiTfWfWgM/giphy.gif)